.travis.yml

```diff
@@ -1,10 +1,13 @@
 language: node_js
 node_js:
   - stable
+cache:
+  directories:
```

Comment:
Based on the article that I linked to I think you need to include `yarn: true` here too.